includes/class-wcs-export-admin.php

```diff
@@ -293,15 +293,26 @@ public function export_crons() {
 					$status = 'processing';
 				}
 
+				$ctime = absint( filectime( trailingslashit( WCS_Exporter_Cron::$cron_dir ) . $file ) );
+
 				$file_data = array(
 					'name'   => $file,
 					'url'    => $files_url . $file,
 					'status' => $status,
-					'date'   => date( 'd/m/Y G:i:s', absint( filectime( trailingslashit( WCS_Exporter_Cron::$cron_dir ) . $file ) ) ),
+					'ctime'  => $ctime,
+					'date'   => date( 'd/m/Y G:i:s', $ctime ),
 				);
 
 				$files_data[] = $file_data;
 			}
+
+			// Sort files by date (desc).
+			usort(
+				$files_data,
+				function( $file1, $file2 ) {
+					return $file2['ctime'] - $file1['ctime'];
+				}
+			);
```

Comment:
Use the [spaceship operator (available since PHP 7.0)](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.spaceship-op):
```suggestion
			usort( $files_data, function( $file1, $file2 ) {
				return $file2['ctime'] <=> $file1['ctime'];
			});
```

In addition, use [arrow functions (available since PHP 7.4)](https://www.php.net/manual/en/functions.arrow.php):
```suggestion
			usort( $files_data, fn( $file1, $file2 ) => $file2['ctime'] <=> $file1['ctime']);
```

[WordPress requires PHP 7.4](https://wordpress.org/about/requirements/), so this is fine.